R/cts.R

```diff
@@ -89,14 +98,32 @@ cts_compinfo <- function(inchikey, verbose = TRUE){
 #' comp <- c("triclosan", "hexane")
 #' cts_convert(comp, "Chemical Name", "cas")
 #' }
-cts_convert <- function(query, from, to, first = FALSE, choices = NULL, verbose = TRUE, ...){
-  if(!missing("first"))
-    stop('"first" is deprecated.  Use "choices = 1" instead.')
+cts_convert <- function(query,
+                        from,
+                        to,
+                        match = c("all", "first", "ask", "na"),
+                        verbose = TRUE,
+                        choices = NULL,
+                        ...){
+  if(!missing("choices"))
+    stop('"choices" is deprecated.  Use "match" instead.')
```

Comment:
Previously deprecated arguments resulted in a warning message but this results in an error. Is there a reason for this?

Reply:
`choices = 10`, for example, doesn't have an exactly equivalent value for `match`. But now that you mention it, I could actually make this a message.
```suggestion
   if(!missing("choices")) {
    if (is.null(choices)) {
      message('"choices" is deprecated.  Using match = "all" instead.')
      match <- "all"
    } else if(choices == 1) {
      message('"choices" is deprecated.  Using match= "first" instead.')
      match <- "first"
    } else if ((is.numeric(choices) & choices > 1) | choices == "all") {
      message('"choices" is deprecated.  Using match = "ask" instead.')
      match <- "ask"
    } else {
      message('"choices" is deprecated.  Using match = "all" instead.')
      match <- "all"
    }
  }
```